File: lib/c/multiplicationtable.c
//multiplication table 
#include<stdio.h>
int main()
{
    int num,n,i,re=0;
    printf("enter the number for which multiplication table to be generated\n");
    scanf("%d",&num);
    printf("enter the max limit of the table\n");
    scanf("%d",&n);
    printf("MULTIPLICATION TABLE:\n");
    for(i=0;i<=n;i++)
    {
re=i*num;
printf(" %d * %d =%d\n",num,i,re);
    }
}

File: multiplication_table.py
number = int(input ("Enter the number of which the user wants to print the multiplication table: "))      
# We are using "for loop" to iterate the multiplication n times 
rg = int(input("enter the range"))
print ("The Multiplication Table of: ", number)    
for count in range(1,rg+1):      
   print (number, 'x', count, '=', number * count)   
